ihmc-humanoid-robotics/src/main/java/us/ihmc/humanoidRobotics/communication/controllerAPI/command/HandHybridJointspaceTaskspaceTrajectoryCommand.java

```diff
@@ -31,6 +44,41 @@ public HandHybridJointspaceTaskspaceTrajectoryCommand(RobotSide robotSide, boole
       this.taskspaceTrajectoryCommand.set(taskspaceTrajectoryCommand);
    }
 
+   public HandHybridJointspaceTaskspaceTrajectoryCommand(RobotSide robotSide, boolean forceExecution, SE3TrajectoryControllerCommand taskspaceTrajectoryCommand,
```

Comment:
these messages are not used for now, but will when replaying back or using a planner that can provide the gain trajectories and feedforward trajectories